example/index.html

```diff
@@ -156,6 +163,49 @@
 
     function onAuthData(data) {
         console.log("auth data received: "+data)
+
+        // extract expiry.
+        let expiry = window[namespace].wasmClientGetExpiry();
+        if (!expiry) {
+            document.getElementById('expiry').innerHTML = "No expiry found in macaroon";
+            return {}
+        }
+
+        localStorage.setItem(namespace+":expiry", expiry)
+        document.getElementById('expiry').innerHTML = "Session Expiry: "+new Date(expiry*1000).toLocaleString();
+
+        // extract permissions.
+        let res = window[namespace].wasmClientHasPerms("lnrpc.Lightning.GetInfo")
+        console.log(res)
```

Comment:
nit: add some context to this log message?

Reply:
oops, left over debug log. Removed!